src/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ add_library(picasso ${SOURCES})
 
 target_link_libraries(picasso
   Kokkos::kokkos
-  kokkoskernels
+  Kokkos::kokkoskernels
```

Comment:
Just a note to users that until the KokkosKernels Trilinos build is resolved this will need to be changed back.